smart-contracts/contracts-common/concordium-contracts-common/Cargo.toml

```diff
@@ -40,7 +40,6 @@ optional = true
 version = "0.4.35" # the patch version is necessary since chrono adds new functionality in patch versions
 
 [dependencies.bs58]
-optional = true
```

Comment:
We need to be careful here, since this will now be a dependency of `concordium-std`.
So you should ensure that this works for webassembly, and also that the resulting module size does not increase based on this.

Reply:
It is moved behind the `derive-serde` feature flag now to not affect the smart contract module size.